proto/store/setting.proto

```diff
@@ -33,6 +33,22 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // We support four levels of AlertLevel: INFO, WARNING, and ERROR.
+  enum AnnouncementAlertLevel {
+    ANNOUNCEMENTALERTLEVEL_INFO = 0;
+    ANNOUNCEMENTALERTLEVEL_WARNING = 1;
+    ANNOUNCEMENTALERTLEVEL_ERROR = 2;
+  }
+
+  // The alert level of announcemnt
+  AnnouncementAlertLevel announcement_level = 7;
```

Comment:
Also, instead of creating 3 fields, let's create a separate `message AnnouncementSetting` and put fields inside it